app/src/androidTest/java/com/sdp/movemeet/Chat/MessageHolderTest.java

```diff
@@ -0,0 +1,5 @@
+package com.sdp.movemeet.Chat;
+
+public class MessageHolderTest {
```

Comment:
empty test class ???

Reply:
You're right, this test is in fact useless since we actually don't have a MessageHolder class (we have a MessageViewHolder class).